Sources/Web3Core/KeystoreManager/EthereumKeystoreV3.swift

```diff
@@ -6,6 +6,7 @@
 import Foundation
 import CryptoSwift
 
+// swiftlint:disable cyclomatic_complexity
```

Comment:
Shouldn't we enable it back at the end of the struct?

Reply:
No, not in this case.
Basically whole file requires refactoring, and for now to silence the cyclomatic_complexity issues in this file I've added that comment.
The rule is automatically re-enabled when SwiftLint leaves this file and starts linting the other one.